modules/kargoBidAdapter.js

```diff
@@ -0,0 +1,166 @@
+import * as utils from 'src/utils';
+import {config} from 'src/config';
+import {registerBidder} from 'src/adapters/bidderFactory';
+const BIDDER_CODE = 'kargo';
+const HOST = 'https://krk.kargo.com';
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: [],
+  isBidRequestValid: function(bid) {
+    if (!bid || !bid.params) {
+      return false;
+    }
+    return !!bid.params.placementId;
+  },
+  buildRequests: function(validBidRequests, bidderRequest) {
+    const currency = config.getConfig('currency');
+    const transformedParams = Object.assign({}, {
+      timeout: bidderRequest.timeout,
+      currency: currency,
+      cpmGranularity: 1,
+      cpmRange: {
+        floor: 0,
+        ceil: 20
+      },
+      adSlotIds: utils._map(validBidRequests, bid => bid.params.placementId)
+    }, spec._getAllMetadata());
+    const encodedParams = encodeURIComponent(JSON.stringify(transformedParams));
+    return Object.assign({}, bidderRequest, {
+      method: 'GET',
+      url: `${HOST}/api/v1/bid`,
+      data: `json=${encodedParams}`,
+      currency: currency
+    });
+  },
+  interpretResponse: function(adUnits, bidRequest) {
+    let bids = {};
+    utils._each(bidRequest.bids, bid => bids[bid.params.placementId] = bid);
+    const bidResponses = [];
+    for (let adUnitId in adUnits) {
+      let adUnit = adUnits[adUnitId];
+      bidResponses.push({
+        requestId: bids[adUnitId].bidId,
+        bidderCode: spec.code,
```

Comment:
Our docs are bad here, but... don't send back the `bidderCode` on bids. The `bidderFactory` adds it to you, and this won't work quite right if someone aliases your adapter (which can be done by customers at runtime)